narwhals/_arrow/group_by.py

```diff
@@ -172,7 +172,10 @@ def agg_arrow(
             function_name = remove_prefix(expr._function_name, "col->")
             function_name = POLARS_TO_ARROW_AGGREGATIONS.get(function_name, function_name)
 
-            option = get_function_name_option(function_name)
+            if expr._function_name != "col->len":
+                option = get_function_name_option(function_name)
+            else:
+                option = pc.CountOptions(mode="all")
```

Comment:
would it be cleaner to have
```
function_name, option = POLARS_TO_ARROW_AGGREGATIONS[function_name]
```
, and map to all the options in `POLARS_TO_ARROW_AGGREGATIONS`?